### PR TITLE
Changed to use RIS firehose feed and one process for all prefixes

### LINKED ISSUE #137: Aggregate monitored sub-prefixes into best-effort super-prefixes
**Is your feature request related to a problem? Please describe.**
Instead of opening one taps per prefix we should do some internal smart aggregation to aggregate them in batches.

**Describe the solution you'd like**
Do the aggregation recursively; monitor the super-prefixes, but discard BGP updates for non-configured prefixes. E.g., if 2 /24 and 1 /23 --> aggregate the /24 in 1 /23 (if neighbors) and then if the new /23 and the old /23 are also neighbors build a /22 and end there. Do this only on monitoring and taps.

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

### LINKED ISSUE #138: Limit taps processes
**Is your feature request related to a problem? Please describe.**
Currently when several taps processes are spawned at the same time there is too much CPU load in the
beginning. Try to spawn batches and wait in-between.

**Describe the solution you'd like**
~~Batch-based spawning with time intervals.~~
Limit to one process per tap type

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by slowr**
### Description of PR

We use [firehose RIPE RIS](https://ris-live.ripe.net/manual/#fullstream) and we do prefix filtering on the tap's side.

What component(s) does this PR affect?

- [x] Back-End (Database, Microservices, Containers, etc)
- [ ] Front-End (Flask, API, etc)
- [ ] Docs
- [ ] Build System

Does the PR require changes on other components? If yes, please mark the components:

- [ ] Back-End (Database, Microservices, Containers, etc)
- [ ] Front-End (Flask, API, etc)
- [ ] Docs
- [ ] Build System

### Related Issue

Resolves #138
Resolves #137

### Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Docs update
- [ ] None of the above

### Checklist:

- [x] I have read the **[contributing guide](https://github.com/FORTH-ICS-INSPIRE/artemis/blob/master/CONTRIBUTING.md)** and my code conforms to the guidelines.
- [ ] This change requires a change in the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: backend/core/taps/ripe_ris.py [hunk 1]
```diff
@@ -96,52 +101,45 @@ def normalize_ripe_ris(msg, conf_prefix):
...
-        "moreSpecific": True,
-        "lessSpecific": False,
-        "socketOptions": {"includeRaw": False},
-    }
 
-    ws.send(json.dumps({"type": "ris_subscribe", "data": params}))
+    events = requests.get(
+        "https://ris-live.ripe.net/v1/stream/?format=json&client={}".format(ris_suffix),
+        stream=True,
+    )
```
Comment (vkotronis) [inline backend/core/taps/ripe_ris.py]: Use artemis-{} so that the user-defined suffix comes after

**Comment by vkotronis** [comment]
@slowr could you recheck the build after merging master here? After that, if everything works fine (check if the correct prefixes are examined in the updates, as expected), you can merge.

**Comment by vkotronis** [comment]
@slowr Also please update the CHANGELOG apart from testing and checking.

**Comment by vkotronis** [comment]
@slowr if this has been tested, and you successfully see both announcements and withdrawals for the correct prefixes (the ones that should be monitored), you can merge this PR.

**Comment by vkotronis** [comment]
@slowr do we need anything else for this PR? Is it mergable?

**Comment by slowr** [comment]
@vkotronis yes it's mergeable

**Pull request merged**